ui/src/containers/CreateVolume.js

```diff
@@ -315,6 +314,7 @@ const CreateVolume = (props) => {
   };
 
   const volumeNameRegex = /^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/;
+  const labelNameRegex = /^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$/;
```

Comment:
This is a valid regex for label values, for keys the validation is a bit more complex, please refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set

Reply:
We might need to change the error message to follow this